src/java/main/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -61,6 +61,7 @@
 
     private static boolean standaloneEnabled = true;
     private static boolean reconfigEnabled = false;
+    private static int warnThreshold = 10000;
```

Comment:
I suggest having to turn on logging by default. Won't hurt much as such. People can opt out if they want. Its similar to usual stats collection IMO.